src/main/java/com/spotinst/sdkjava/model/converters/ElastigroupConverterSuspendAzure.java

```diff
@@ -0,0 +1,30 @@
+package com.spotinst.sdkjava.model.converters;
+
+import com.spotinst.sdkjava.model.api.azure.elastiGroup.V3.ApiSuspendgroupProcessesAzure;
+import com.spotinst.sdkjava.model.bl.azure.elastiGroup.V3.SuspendGroupProcessesAzure;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ElastigroupConverterSuspendAzure {
+
+    public static List<ApiSuspendgroupProcessesAzure> toDal(List<SuspendGroupProcessesAzure> suspendGroupProcesses) {
+
+        List<ApiSuspendgroupProcessesAzure> retVal = null;
+        if (suspendGroupProcesses != null) {
```

Comment:
Instead of loop, can you try in a single line like below 
List<ApiLaunchSpecMetadataSpecification> metadata =
                        launchSpec.getMetadata().stream().map(OceanGkeConverter::toDal)
                           .collect(Collectors.toList());
                apiLaunchSpec.setMetadata(metadata);

Reply:
This I have already tried, there was error hence used loop and added in separate class as well.